src/test/java/giis/eshopcontainers/e2e/functional/tests/WebAggAPITests.java

```diff
@@ -0,0 +1,142 @@
+package giis.eshopcontainers.e2e.functional.tests;
+
+import com.google.gson.Gson;
+import giis.eshopcontainers.e2e.functional.common.BaseAPIClass;
+import giis.eshopcontainers.e2e.functional.model.Order;
+import giis.eshopcontainers.e2e.functional.model.OrderItem;
+import giis.retorch.annotations.AccessMode;
+import giis.retorch.annotations.Resource;
+import org.apache.http.HttpEntity;
+import org.apache.http.HttpResponse;
+import org.apache.http.client.HttpClient;
+import org.apache.http.client.ResponseHandler;
+import org.apache.http.client.methods.HttpGet;
+import org.apache.http.client.methods.HttpPost;
+import org.apache.http.entity.StringEntity;
+import org.apache.http.impl.client.BasicResponseHandler;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.impl.client.HttpClients;
+import org.apache.http.util.EntityUtils;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+import java.util.List;
+
+public class WebAggAPITests extends BaseAPIClass {
```

Comment:
I dont get the meaning of WebAgg, could you chose a better name? and some comment here would be useful.